### ISSUE TITLE
[BUG] 'Min Burst:' setting disappears when opening and closing the help menu  

### ISSUE THREAD

**Issue by IliyaFrolov** [kraanzu/smassh#29]
**Describe the bug** 🐛
The 'Min Burst:' setting within the 'Push Your Limits' menu disappears when toggling the help menu from a different settings menu. The missing setting can be made to reappear by toggling the help menu once again but from the 'Push Your Limits' menu itself.

**To Reproduce** 🐣
Steps to reproduce the behaviour:
1. Within 'Settings', go to any menu other than the 'Push your Limits' menu.
2. Press 'ctrl+h' twice to open and close the help menu.
3. Go to the 'Push your Limits' menu. By this point, the setting 'Min Burst:' should be gone.
4. Repeat step (2.). The setting should reappear.

**Expected behaviour**  🤔 
The settings should not disappear and reappear when opening and closing the help menu.

**Screenshots** 🧐
Before: https://user-images.githubusercontent.com/68474507/180227692-93727a56-67a5-40c5-ac46-b163cda21380.png
After: https://user-images.githubusercontent.com/68474507/180227924-bd8a5cde-5cd0-4b01-811d-0dc014b6b715.png

**Desktop (please complete the following information):**  🤖
 - OS: Windows 11
 

**Additional context** 📝

**Comment by IliyaFrolov** [comment]
As a side note, I'm pretty new when it comes to programming and especially working on big projects. As such, I'd like to gain more experience by contributing to projects like this one. Would I be able to give a shot at solving this bug? It may be something very simple and easy to fix, but at the very least it'll be something, and understanding the codebase is valuable as well.

**Comment by kraanzu** [comment]
Hey @IliyaFrolov , 
Thanks for the detailed explanation. I really appreciate it :)
I cannot reproduce the behavior on my machine but i am certain that it is related to the size of panel of the help menu that's being rendered

**Comment by kraanzu** [comment]
>  I'm pretty new when it comes to programming and especially working on big projects

Hey it's not biggie...we all start at something..I too haven't really contributed to projects other than my own :)

> I'd like to gain more experience by contributing to projects like this one

Sure go ahead <3

> Would I be able to give a shot at solving this bug?

As I said before, it's just due to different sizes being rendered which caused the confusion for the bottom area to update accordingly and hence the issue (I will have to open an issue on textual later)

> and understanding the codebase is valuable experience as well I think.

Right! Ping me if you need any pointers. I'll be more than happy to help <3

**Comment by IliyaFrolov** [comment]
> As I said before, it's just due to different sizes being rendered which caused the confusion for the bottom area to update accordingly and hence the issue (I will have to open an issue on textual later)

Ah I see, that would make sense. Thank you for the explanation!

> Right! Ping me if you need any pointers. I'll be more than happy to help <3

Thanks! If I get stuck, I'll be sure to give you a message.

**Comment by kraanzu** [comment]
Hey @IliyaFrolov,
You didn't make a PR. Are you not fixing this?

**Comment by IliyaFrolov** [comment]
> Hey @IliyaFrolov, You didn't make a PR. Are you not fixing this?

Ah sorry, I think I must've misunderstood you. I assumed you were taking control of the issue after you said you were going to open an issue on textual later (which I didn't exactly know what this meant, but I made the assumption anyway), so that's my bad. 

I'll reopen the bug and will try fixing it based on the information you gave before. Just as a heads up though in case this is something you wanted a fix for quickly, I'll be away on camping today till Sunday, so may not get round to fixing the bug until I'm back.

**Comment by kraanzu** [comment]
> you were going to open an issue on textual later

My bad. I thought maybe it's something i am doing wrong... So I'll check what's causing the problem and will open a issue . \
In the meantime... a temporary fix would be to increase the height of the help-menu panel :)

**Comment by IliyaFrolov** [comment]
> My bad. I thought maybe it's something i am doing wrong... So I'll check what's causing the problem and will open a issue . In the meantime... a temporary fix would be to increase the height of the help-menu panel :)

Okay, sounds good to me!

**Comment by somenothing** [comment]
@IliyaFrolov I tested it with both Powershell and Windows Terminal in Windows 11, but this bug didn't happen. You can try termtyper's latest version by re-run `python3 -m pip install git+https://github.com/kraanzu/termtyper.git` or find exe files in Acions.

**Comment by IliyaFrolov** [comment]
> @IliyaFrolov I tested it with both Powershell and Windows Terminal in Windows 11, but this bug didn't happen. You can try termtyper's latest version by re-run `python3 -m pip install git+https://github.com/kraanzu/termtyper.git` or find exe files in Acions.

Yeah I'm a bit confused now because I just tried to use termtyper on my pc (I used my laptop before) and it was flickering on first launch, then every consecutive attempt resulted in a blank blue terminal. I used powershell with windows 10.

So now I'm thinking perhaps both of these issues are on my end rather than being bugs, but I'm unsure.

**Comment by kraanzu** [comment]
> it was flickering on first launch, then every consecutive attempt resulted in a blank blue terminal

Yup.. That's windows doing this

> both of these issues are on my end rather than being bugs, but I'm unsure.

The `Min Burst` disappearance that you mentioned is due to some specific resolutions... Maybe try changing that?

**Comment by kraanzu** [comment]
This works differently now so I'll close this!
Feel free to test out the latest ver and open a new issue if the problem still persists :D
